maddymz/irondb
# Help Page

To Test:

1. git checkout HelpPage
2. ./iron.sh -lepm

Updates to the Help Page:
3. Check documentation on the public Help page
4. Log in and check documentation on the user Help Page
5. If you like both Help pages, we good.

New Registration Alerts:
6. Navigate to login page and enter an incorrect user and/or password.
**Expected Alert:**
![Screen Shot 2019-04-26 at 5 43 16 PM](https://user-images.githubusercontent.com/6512755/56838157-02c7df80-684b-11e9-921c-cd177b0445f0.jpg)

7. Register a new user.
**Expected Alert:**
![Screen Shot 2019-04-26 at 5 42 05 PM](https://user-images.githubusercontent.com/6512755/56838140-f6438700-684a-11e9-9f40-633bfb5163d6.jpg)
